code/modules/mob/living/carbon/life.dm

```diff
@@ -336,7 +336,8 @@
 	if(stat != DEAD)
 		for(var/V in internal_organs)
 			var/obj/item/organ/O = V
-			O.on_life()
+			if(O.owner) // This exist mostly because reagent metabolization can cause organ reshuffling

```

Comment:
I'm not sure if i should bother checking if the mob is the same one in case someone adds liquid organ transplant reagent..

Reply:
Sounds like a problem for future AnturK :smirk: